ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -464,6 +464,12 @@ def __array__(self, dtype=None, /):
         if not dtype:
             return self
         return np_frontend.array(self, dtype=dtype)
+    
+    def __array_wrap__(self, out_arr, context=None):
+        if context is None:
+            return np_frontend.asarray(self)
```

Comment:
`ndarray` has got a `view` method you can use.

Reply:
Ok, ma'am,